kolibri/plugins/audio_mp3_render/kolibri_plugin.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+"""
+Copy and modify this code for your own plugin.
+"""
+from __future__ import absolute_import, print_function, unicode_literals
+
+import logging
+
+from kolibri.plugins.base import KolibriFrontEndPluginBase
+from kolibri.plugins.hooks import CONTENT_RENDERER_ASYNC, FRONTEND_PLUGINS
+
+logger = logging.getLogger(__name__)
+
+
+class KolibriAudioMP3RenderFrontEnd(KolibriFrontEndPluginBase):
+    """
+    The base learn code for the learn page.
+    """
+    entry_file = "assets/src/render_audio_mp3_module.js"
+
+    events = {
+        "content_render:audio/mp3": "render"
```

Comment:
I can't think of a use case for it immediately - but there may be a case where we might want to register `content_render:*` (or some other catch all - if we're going to stick with Backbone style events, then maybe triggering `content_render:audio/mp3` also automatically triggers `content_render`) - and then that will trigger the content recommendation to load relevant content that a learner might want to navigate to next.

In this case, I think it is fine to have multiple event listeners - and it seems like idiomatic Javascript too. What is less clear is if two different content playing plugins have both been registered at once. Do we want to legislate for that?